db.json

```diff
@@ -0,0 +1,59 @@
+{
+  "indicator": [
+    {
+      "id": 1,
+      "name": "Indicator1",
+      "query": "",
+      "slug": "indicator-1",
+      "category": "wildlife",
+      "threshold": 1,
```

Comment:
We need to discuss the type of values that you are using here. I think that `threshold` should be an ```array```... and type a ```string```